src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.MaxResponseHeadersLength.cs

```diff
@@ -45,16 +45,31 @@ public void ValidValue_SetGet_Roundtrips(int validValue)
         }
 
         [Fact]
-        [OuterLoop("Uses external server")]
         public async Task SetAfterUse_Throws()
         {
-            using (HttpClientHandler handler = CreateHttpClientHandler())
-            using (HttpClient client = CreateHttpClient(handler))
+            await LoopbackServer.CreateClientAndServerAsync(async uri =>
             {
-                handler.MaxResponseHeadersLength = 1;
-                (await client.GetStreamAsync(Configuration.Http.RemoteEchoServer)).Dispose();
-                Assert.Throws<InvalidOperationException>(() => handler.MaxResponseHeadersLength = 1);
-            }
+                using (HttpClientHandler handler = CreateHttpClientHandler())
+                using (HttpClient client = CreateHttpClient(handler))
+                {
+                    client.BaseAddress = uri;
```

Comment:
nit: it's not really necessary to use the `.BaseAddress` property in this test. In almost all our tests, we simply pass the uri to the HttpClient method itself.

Reply:
Changed. Did this to avoid using `.ToString()`.